packages/ai-jsx/src/lib/asr/asr.tsx

```diff
@@ -166,9 +177,10 @@ export type GetTokenFunction = (provider: string) => Promise<string>;
  * between when VAD detected silence and the transcript was received.
  */
 export class Transcript {
-  constructor(public text: string, public final: boolean, public latency?: number) {
+  constructor(public text: string, public final: boolean, public timestamp: number, public latency?: number) {
     this.text = text;
     this.final = final;
+    this.timestamp = timestamp;
     this.latency = latency;
```

Comment:
I feel like none of this should be necessary. I think the `public text: string` in the constructor params is syntactic sugar for this.